### DISCUSSION TITLE
Nintendo Music Support

### DISCUSSION THREAD

**Discussion by beeldubz** [music-assistant/support#3122] [Feature requests and ideas]
I'm not sure if it's even possible but it would be nice to have my nintendo music library to play through music assistant. It is fairly new but it does use airplay. Chromecast not supported yet.  I know nintendo isn't the most open source friendly though too.

**Comment by dscarr10** [comment]
I would also highly appreciate Nintendo music getting added as a provider. I have a family account with open spots that I can use for anyone that needs access to be able to work on this.

**Comment by dlutton** [comment]
I'd also like this to be a provider if possible.  Thank you!

**Comment by OisinNH** [comment]
Whilst I am no expert (or even amateur), I imagine it may now be possible; Nintendo, today, released a [web player](https://music.nintendo.com/) for the platform.

**Reply by dscarr10** [reply]
This would be awesome if true. Access to a family plan for someone that is willing to work on the integration is still being offered by me.

**Comment by DeanDotDan** [comment]
This would be a great new provider!
